### DISCUSSION TITLE
How Do I create the extra layer of security?

### DISCUSSION THREAD

**Discussion by Wm-M-Thompson** [community/community#36821] [Code Security] [answered]
I have already done this but only once and now I have forgotten about it.  When I try to upload content from the command line, for the first time, I enter:

- git init
- git remote add origin "http:/github.com/xarxu/[NAME OF REPOSITORY].git"
- git remote -v
- git add .
- git commit -m "first commit of..."
- git push origin master

And this si where I get a prompte for username and password.  But if I use the password, I am told that github has a new program now for security that is more secure than a password.  It says

"remote: Support for password authentication was removed on August 13, 2021."
"remote" Please see..."

And then I am given a url for doc.github.com that does not provide the right information.

I think the information is wrong because the title on the page, "Cloning with HTTPS URLs" and not something like, "new security method that replaces passwords for remote command line github on ubuntu"

Please advise.  Please help.
- 

-

**Comment by paescuj** [comment, marked as answer]
As stated in the [docs](https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls) you have to create and use a personal access token instead of the password.

**Reply by Wm-M-Thompson** [reply]
https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token

**Reply by kude5n1k** [reply]
that was in my answer

**Comment by kude5n1k** [comment]
Hi friend!
You need to create a personal access [token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
